linux_os/guide/system/permissions/files/file_permissions_etc_audit_rulesd/rule.yml

```diff
@@ -0,0 +1,36 @@
+documentation_complete: true
+
+prodtype: fedora,rhel8
+
+title: 'Verify Permissions on /etc/audit/rules.d/*.conf'
+
+description: |-
+    {{{ describe_file_permissions(file="/etc/audit/rules.d/*.conf", perms="0640") }}}
+
+
+rationale: |-
+    Without the capability to restrict the roles and individuals that can select which events
+    are audited, unauthorized personnel may be able to prevent the auditing of critical
+    events. Misconfigured audits may degrade the system's performance by overwhelming
+    the audit log. Misconfigured audits may also make it more difficult to establish,
+    correlate, and investigate the events relating to an incident or identify
+    those responsible for one.
+
+severity: medium
+
+identifiers:
+    cce@rhel8: CCE-85875-3
+
+references:
+    nist: AU-12b
+    disa: CCI-000171
+    srg: SRG-OS-000063-GPOS-00032
+    stigid@rhel8: RHEL-08-030610
+
+template:
+    name: file_permissions
+    vars:
+      filepath: /etc/audit/rules.d/
+      file_regex: ^.*\\.conf$
+      allow_stricter_permissions: "true"
+      filemode: '0640'
```

Comment:
```suggestion
title: 'Verify Permissions on /etc/audit/rules.d/*.rules'

description: |-
    {{{ describe_file_permissions(file="/etc/audit/rules.d/*.rules", perms="0640") }}}


rationale: |-
    Without the capability to restrict the roles and individuals that can select which events
    are audited, unauthorized personnel may be able to prevent the auditing of critical
    events. Misconfigured audits may degrade the system's performance by overwhelming
    the audit log. Misconfigured audits may also make it more difficult to establish,
    correlate, and investigate the events relating to an incident or identify
    those responsible for one.

severity: medium

identifiers:
    cce@rhel8: CCE-85875-3

references:
    nist: AU-12b
    disa: CCI-000171
    srg: SRG-OS-000063-GPOS-00032
    stigid@rhel8: RHEL-08-030610

template:
    name: file_permissions
    vars:
      filepath: /etc/audit/rules.d/
      file_regex: ^.*\\.rules$
      allow_stricter_permissions: "true"
      filemode: '0640'
```

Reply:
We should be checking the *.rules files in this directory, not *.conf. Otherwise looks good.